nuxeo-core/nuxeo-core-event/src/test/resources/OSGI-INF/test-scheduler-with-delay-config.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.core.scheduler.test.with.delay.config">
+
+  <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+    <property name="org.nuxeo.scheduler.start.delay">10</property>
+  </extension>
+
+  <extension
+    target="org.nuxeo.ecm.core.scheduler.SchedulerService"
+    point="schedule">
+    <schedule id="testing">
+      <username>Administrator</username>
+      <eventId>testEvent</eventId>
+      <eventCategory>default</eventCategory>
+      <!-- every second -->
+      <cronExpression>*/1 * * * * ?</cronExpression>
+    </schedule>
+  </extension>
+</component>
```

Comment:
There's missing a new line at the end of the file.

Reply:
changed